x/tokenfactory/keeper/createdenom_test.go

```diff
@@ -51,6 +51,10 @@ func (suite *KeeperTestSuite) TestMsgCreateDenom() {
 }
 
 func (suite *KeeperTestSuite) TestCreateDenom() {
+	largeAllowList := make([]string, 2001)
+	for i := 0; i < 2001; i++ {
```

Comment:
Nit: Should we make this size (2001) relative to the MaxAllowListSize stored in params? (Ie MaxAlowListSize + 1).

Reply:
done